packages/recommend-core/src/utils/getSearchParametersForModel.ts

```diff
@@ -4,47 +4,47 @@ import { GetRecommendationsInternalProps } from '../getRecommendations';
 import { RecommendModel, RecommendRecord } from '../types';
 
 type GetSearchParametersParams<TObject> = {
-  fallbackFilters: GetRecommendationsInternalProps<TObject>['fallbackFilters'];
+  fallbackParameters: GetRecommendationsInternalProps<TObject>['fallbackParameters'];
   recommendations: RecommendRecord[];
   threshold: GetRecommendationsInternalProps<TObject>['threshold'];
 };
 
 function getFiltersFromRecommendations<TObject>({
-  fallbackFilters,
+  fallbackParameters,
   recommendations,
   threshold,
 }: GetSearchParametersParams<TObject>): SearchOptions['optionalFilters'] {
   if (recommendations.length === 0) {
-    return fallbackFilters;
+    return fallbackParameters.facetFilters;
   }
 
   const recommendationFilters = recommendations
     .filter((recommendation) => recommendation.score > threshold)
     .map(({ objectID, score }) => `objectID:${objectID}<score=${score * 100}>`);
 
-  return [...recommendationFilters, ...fallbackFilters];
+  return [...recommendationFilters, ...fallbackParameters.facetFilters!];
```

Comment:
This could also probably throw if `facetFilters` is not defaulted.

```suggestion
  const facetFilters = fallbackParameters.facetFilters || []

  if (recommendations.length === 0) {
    return facetFilters;
  }

  const recommendationFilters = recommendations
    .filter((recommendation) => recommendation.score > threshold)
    .map(({ objectID, score }) => `objectID:${objectID}<score=${score * 100}>`);

  return [...recommendationFilters, ...facetFilters];
```